easybuild/easyconfigs/l/libyaml/libyaml-0.2.5-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'libyaml'
+version = '0.2.5'
+
+homepage = 'https://pyyaml.org/wiki/LibYAML'
+description = "LibYAML is a YAML parser and emitter written in C."
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://pyyaml.org/download/%(name)s/']
+sources = ['yaml-%(version)s.tar.gz']
+checksums = ['c642ae9b75fee120b2d96c712538bd2cf283228d2337df2cf2988e3c02678ef4']
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+
+sanity_check_paths = {
+    'files': ['include/yaml.h', 'lib/%(name)s.a', 'lib/%(name)s.so'],
```

Comment:
@casparvl please restore the handling of shared library file extensions with `SHLIB_EXT`.

Reply:
Done. Silly `--try-toolchain-version`  seems to have resolved it... :)